ArcGISRuntimeSDKQt_CppSamples/Scenes/AnimateImagesWithImageOverlay/README.md

```diff
@@ -0,0 +1,38 @@
+# Animate images with image overlay
+
+Animate a series of images with an image overlay.
+
+![](screenshot.png)
+
+## Use case
+
+An image overlay is useful for displaying fast and dynamic images; for example, rendering real-time sensor data captured from a drone. Each frame from the drone becomes a static image which is updated on the fly as the data is made available.
```

Comment:
I just realized this now, but since we decided on using the radar use case for the sample, it is a bit strange to use the drone use case in the readme and then the actual sample shows a radar use case.